test/v1/controllers/validators/DeleteOtherReliefsValidatorFactorySpec.scala

```diff
@@ -1,79 +0,0 @@
-/*
- * Copyright 2023 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package v1.controllers.validators
-
-import api.models.domain.{Nino, TaxYear}
-import api.models.errors._
-import support.UnitSpec
-import v1.models.request.deleteOtherReliefs.DeleteOtherReliefsRequestData
-
-class DeleteOtherReliefsValidatorFactorySpec extends UnitSpec {
```

Comment:
This should not have been deleted. It should be restored (as `Def1_DeleteOtherReliefsValidatorSpec` under def1 package and is should test the validator now (not the validator factory).